src/main/java/fittrack/storage/Storage.java

```diff
@@ -0,0 +1,108 @@
+package fittrack.storage;
+
+import fittrack.trainingsession.TrainingSession;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import static fittrack.logger.FitTrackLogger.LOGGER;
+
+public class Storage {
+
+    public static final String SAVE_FILE = "data/saveFile.txt"; // Path to the save file
+    public static final File SAVEFILE = new File(SAVE_FILE); // File object for the save file
```

Comment:
Consider using more meaningful variable names, SAVE_FILE and SAVEFILE does not provide much context as to what differentiates them, leading to confusion when using it in the code